src/main/java/com/xpandit/plugins/xrayjenkins/task/filefilters/OnlyFeatureFilesInPathFilter.java

```diff
@@ -0,0 +1,60 @@
+package com.xpandit.plugins.xrayjenkins.task.filefilters;
+
+import com.xpandit.plugins.xrayjenkins.exceptions.XrayJenkinsGenericException;
+import hudson.FilePath;
+import java.io.File;
+import java.io.FileFilter;
+import java.io.IOException;
+import java.util.Date;
+import java.util.Set;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.time.DateUtils;
+
```

Comment:
Please add javadoc explaining the goal of this class, the criteria of the filter, etc